File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnFieldDecimal.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldDecimal extends DefnFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public Double defaultValue;

  @Nullable
  public Double defaultVar;

  @Nullable
  public Double max;

  @Nullable
  public MetaIdField maxFieldId;

  @Nullable
  public Double maxVar;

  @Nullable
  public Double min;

  @Nullable
  public Double minDisplayValue;

  @Nullable
  public MetaIdField minFieldId;

  @Nullable
  public Double minVar;

  @Nullable
  public String numberFormat;

  @Nullable
  public Long numberOfDigitsAfterPeriod;

  @Nullable
  public MetaIdField numberOfDigitsAfterPeriodFieldId;

  @Nullable
  public Long numberOfDigitsAfterPeriodVar;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldDecimal.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdField

interface DefnFieldDecimal : DefnFieldEditable {
    val defaultFieldId: MetaIdField?
    val defaultValue: Double?
    val defaultVar: Double?
    val max: Double?
    val maxFieldId: MetaIdField?
    val maxVar: Double?
    val min: Double?
    val minDisplayValue: Double?
    val minFieldId: MetaIdField?
    val minVar: Double?
    val numberFormat: String?
    val numberOfDigitsAfterPeriod: Double?
    val numberOfDigitsAfterPeriodFieldId: MetaIdField?
    val numberOfDigitsAfterPeriodVar: Double?
}
